crates/bevy_text/src/text.rs

```diff
@@ -170,3 +173,28 @@ impl Default for TextStyle {
         }
     }
 }
+
+/// Determines how lines will be broken when preventing text from running out of bounds.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Reflect, Serialize, Deserialize)]
+#[reflect(Serialize, Deserialize)]
+pub enum TextLineBreakBehaviour {
```

Comment:
Bikeshed: Could this be renamed to `BreakLineOn`? I find `TextLineBreakBehaviour` to be fairly verbose. Specifically the "Behavior" here feels redundant.

Reply:
That's a much better name. I'll do that.